addons/sale/models/sale.py

```diff
@@ -676,18 +676,26 @@ def action_confirm(self):
     def _get_forbidden_state_confirm(self):
         return {'done', 'cancel'}
 
+    def _prepare_analytic_account_data(self, prefix=None):
```

Comment:
Need to seperate PR's I would have thought. Pretty rare for a PR doing 2 unrelated things to 2 modules to be accepted but see how you go.

Reply:
I was on my way to create more commits to improve extendability but just stopped to see if I would get my luck.